src/app/zap-templates/common/ChipTypesHelper.js

```diff
@@ -15,36 +15,35 @@
  *    limitations under the License.
  */
 
-function asBasicType(type)
-{
-  switch (type) {
-  case 'chip::ActionId':
-  case 'chip::FabricIndex':
-    return 'uint8_t';
-  case 'chip::EndpointId':
-  case 'chip::GroupId':
-  case 'chip::VendorId':
-    return 'uint16_t';
-  case 'chip::ClusterId':
-  case 'chip::AttributeId':
-  case 'chip::FieldId':
-  case 'chip::EventId':
-  case 'chip::CommandId':
-  case 'chip::TransactionId':
-  case 'chip::DeviceTypeId':
-  case 'chip::StatusCode':
-  case 'chip::DataVersion':
-    return 'uint32_t';
-  case 'chip::EventNumber':
-  case 'chip::FabricId':
-  case 'chip::NodeId':
-    return 'uint64_t';
-  default:
-    return type;
-  }
+function asBasicType(type) {
```

Comment:
could we separate a javascript restyle from the ZAP updates (that can cause size changes or zap specific changes)?

This PR is hard to review when  mixing restyle with functionality. Restyle changes should be more trivial to approve and merge.

Reply:
Good point. I'll keep these separated in the future and push just the style changes as a separate commit.